packages/widgets/src/components/bubble/bubble-chart.widget.stories.tsx

```diff
@@ -39,12 +61,40 @@ export const widget = () => (
         ),
       }}
       card={cardKnobs('Card')}
+      legend={{
+        enabled: boolean('Enabled', true, 'Legend'),
+        position: select(
+          'Position',
+          positionOptions,
+          positionOptions.left,
+          'Legend'
+        ) as any,
+        alignment: select(
+          'Alignment',
+          alignmentOptions,
+          alignmentOptions.right,
+          'Legend'
+        ),
+        typography: typographyKnobs('Legend', { fontSize: 10 }),
+      }}
+      legendTitle={{
```

Comment:
Rather than creating a separate `legendTitle` property - please extend the `legend` with optional title field.